Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -101,7 +101,8 @@ protected Gradient GetLineGradient(TeleportSurfaceResult targetResult)
         #region IMixedRealityPointer Implementation
 
         /// <inheritdoc />
-        public override bool IsInteractionEnabled => !IsTeleportRequestActive && teleportEnabled;
+        public override bool IsInteractionEnabled => !IsTeleportRequestActive && teleportEnabled
+            && (MixedRealityToolkit.IsInitialized && MixedRealityToolkit.HasActiveProfile && MixedRealityToolkit.Instance.ActiveProfile.IsTeleportSystemEnabled);
```

Comment:
This line basically means "if teleportation system is online" -> suggestion is to make some other helper bool (i.e. private bool IsTeleportationSystemPresent) because there are two instances of these three specific things you have to reference at the same time (and it looks like in the same way, since you need to call inited first, then active profile, then teleportsystemenabled)

Reply:
Yeah, was thinking of doing that, will do that